resources/benchmark-runner.mjs

```diff
@@ -211,33 +211,28 @@ export class BenchmarkRunner {
                 await this._client.willRunTest(suite, test);
 
             setTimeout(() => {
-                this._runTest(suite, test, this._page, async (syncTime, asyncTime) => {
-                    const suiteResults = this._measuredValues.tests[suite.name] || { tests: {}, total: 0 };
-                    const total = syncTime + asyncTime;
-                    this._measuredValues.tests[suite.name] = suiteResults;
-                    suiteResults.tests[test.name] = { tests: { Sync: syncTime, Async: asyncTime }, total: total };
-                    suiteResults.total += total;
-
-                    if (this._client?.didRunTest)
-                        await this._client.didRunTest(suite, test);
-
-                    resolve();
-                });
+                this._runTest(suite, test, this._page, resolve);
             }, 0);
         });
     }
 
     // This function ought be as simple as possible. Don't even use Promise.
-    _runTest(suite, test, page, callback) {
-        performance.mark(`${suite.name}.${test.name}-start`);
+    _runTest(suite, test, page, test_done_callback) {
```

Comment:
Please use camelCase for testDoneCallback.

Reply:
argh, sorry. Too much python 😛